src/bilder/images/edxapp/templates/common_values.yml

```diff
@@ -333,7 +332,7 @@ FOOTER_ORGANIZATION_IMAGE: images/logo.png
 GITHUB_REPO_ROOT: /edx/var/edxapp/data
 GOOGLE_ANALYTICS_ACCOUNT: G-1EW36DF67W
```

Comment:
Shouldn't this have a key also as it'll be different for prod?

Reply:
Yes, I still need to decide how we want to handle setting values for things that aren't directly driven by infrastructure elements.